addon/-private/execution_context/integration.js

```diff
@@ -33,7 +33,9 @@ IntegrationExecutionContext.prototype = {
   },
 
   // Do nothing in integration test
-  visit: $.noop,
+  visit() {
+    throw new Error('visit is not supported in integration mode');
```

Comment:
should I rollback this?

Reply:
I've tried to use `throwBetterError` here. Unfortunately it doesn't work out of the box because `throwBetterError` is currently coupled elusively to element find failures.

 This pr is already quite large. I've reverted original silent behavior.

Reply:
For future reference, here's an example of where `throwBetterError` is used outside the context of element find failures: https://github.com/san650/ember-cli-page-object/blob/master/addon/-private/properties/alias.js#L86.